skimage/transform/radon_transform.py

```diff
@@ -13,16 +13,22 @@
 __all__ = ['radon', 'order_angles_golden_ratio', 'iradon', 'iradon_sart']
 
 
-def radon(image, theta=None, circle=True, *, preserve_range=False):
+def _get_default_center(image_shape, legacy: bool = True):
```

Comment:
Yes, when adding type hints, the convention changes. This can be seen for instance here: https://docs.python.org/3/library/typing.html#typing.Optional
Moreover, `black` systematically adds the spaces in there.

Reply:
```suggestion
def _get_default_center(image_shape, legacy=True):
```
We are currently not using type hints. We are planning to but those wouldn't be inline. So this lonely type hint should be removed. :smile:

Reply:
@Obi-Wan good to know, thanks! Please ignore my suggestion and, for this PR, include @lagru's.